src/ui/component/AddressList/index.tsx

```diff
@@ -46,62 +55,79 @@ const AddressList: any = forwardRef(
         addressItems.current[group.type] = new Array(group.accounts.length);
       }
     });
-
     const updateAllBalance = () => {
       const q: Promise<void>[] = [];
       Object.values(addressItems.current).forEach((arr: any) => {
-        q.push(...arr.map((el) => el.updateBalance()));
+        q.push(...arr.filter(Boolean).map((el) => el.updateBalance()));
```

Comment:
because using the react-window components, only the loaded or in view elements can have updatebalance